src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -276,4 +276,4 @@ internal static string ParseLanguage(string scriptFilePath, IEnumerable<WorkerCo
             return null;
         }
     }
-}
```

Comment:
nit: undo whitespace change in this PR (since this PR is already pretty big, let's scope to the required changes)

Reply:
missed to include a change in this file. Next iteration will handle that.